recurly/notifications/notifications_test.go

```diff
@@ -0,0 +1,178 @@
+package notifications_test
+
+import (
+	"encoding/xml"
+	"net/http"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/blacklightcms/go-recurly/recurly"
+	"github.com/blacklightcms/go-recurly/recurly/notifications"
+)
+
+func TestParse_SuccessfulPaymentNotification(t *testing.T) {
+	xmlFile, err := os.Open("xml/successful_payment_notification.xml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer xmlFile.Close()
+
+	req, err := http.NewRequest("POST", "/", xmlFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	result, err := notifications.Parse(req.Body)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var n *notifications.SuccessfulPaymentNotification
+	var ok bool
+	switch result.(type) {
+	case *notifications.SuccessfulPaymentNotification:
+		n, ok = result.(*notifications.SuccessfulPaymentNotification)
+		if !ok {
+			t.Fatalf("unable to reflect interface")
+		}
+		break
+	default:
+		t.Fatalf("unexpected result type: %T", result)
+	}
```

Comment:
Removed in [910c54](https://github.com/blacklightcms/go-recurly/pull/6/commits/15fba61ea259a3c8441c3a2f23a20c5319910c54)